README.md

```diff
@@ -73,15 +73,16 @@ To run the server in debug mode set:
 
     export FEC_WEB_DEBUG=true
 
-To use styles served from a custom location (e.g., if developing against a local `fec-style`):
+### Developing with fec-style (optional)
+If you're developing with a local instance of [fec-style](https://github.com/18F/fec-style) and want to pull in styles and script changes as you go, use `npm link` to create a symbolic link to your local fec-style repo:
 
-    export FEC_WEB_STYLE_URL=http://localhost:8080/css/styles.css
+    npm link fec-style > ~/[path to fec-style]/fec-style
```

Comment:
You don't actually want that `>` in the command, do you?

The steps that I'm used to are as per https://docs.npmjs.com/cli/link

```
cd ~/fec-style
npm link
cd ~/openFEC-web-app
npm link fec-style
```